### PR TITLE
PoiDetail

### PR DESCRIPTION
**Pull request by jcaliksygic**
AddressCell renamed to HeaderCell,
VC presentAsChildViewController for better understanding,
doc,
DataSources instead of ViewModels

### CODE REVIEW

File: SygicUIKit/Classes/Views/BottomSheets/PoiDetail/PoiDetailHeaderCell.swift [hunk 1]
```diff
@@ -55,7 +55,7 @@ public class PoiDetailAddressCell: UITableViewCell {
     }
 
  // MARK: - Public Methods
-    public func update(with viewModel: SYUIPoiDetailAddressDataSource) {
+    public func update(with viewModel: SYUIPoiDetailHeaderDataSource) {
```
Comment (marekdovjak) [inline SygicUIKit/Classes/Views/BottomSheets/PoiDetail/PoiDetailHeaderCell.swift]: viewModel premenovat

File: SygicUIKit/Classes/Views/BottomSheets/PoiDetail/SYUIPoiDetailView.swift [hunk 2]
```diff
@@ -51,8 +50,8 @@ internal class SYUIPoiDetailView: UIView {
     }
     
     public func reloadData() {
-        guard let viewModel = viewModel else { return }
-        actionButtonsView.update(with: viewModel.buttonsViewModel)
+        guard let viewModel = delegate else { return }
```
Comment (marekdovjak) [inline SygicUIKit/Classes/Views/BottomSheets/PoiDetail/SYUIPoiDetailView.swift]: premenovat viewModel

File: SygicUIKit/Classes/Views/BottomSheets/PoiDetail/SYUIPoiDetailView.swift [hunk 3]
```diff
@@ -137,24 +136,24 @@ extension SYUIPoiDetailView: UITableViewDataSource {
     }
     
     public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        guard let viewModel = viewModel, let section = SYUIPoiDetailSectionType(rawValue: section) else { return 0 }
-        if section == .address {
+        guard let viewModel = delegate, let section = SYUIPoiDetailSectionType(rawValue: section) else { return 0 }
```
Comment (marekdovjak) [inline SygicUIKit/Classes/Views/BottomSheets/PoiDetail/SYUIPoiDetailView.swift]: viewmodel...

File: SygicUIKit/Classes/Views/BottomSheets/PoiDetail/SYUIPoiDetailView.swift (hunk 2 above)
Comment (jcaliksygic) [reply SygicUIKit/Classes/Views/BottomSheets/PoiDetail/SYUIPoiDetailView.swift]: premenoval som tusim v brenchi k action buttonom

File: SygicUIKit/Classes/Views/BottomSheets/PoiDetail/SYUIPoiDetailView.swift (hunk 3 above)
Comment (jcaliksygic) [reply SygicUIKit/Classes/Views/BottomSheets/PoiDetail/SYUIPoiDetailView.swift]: premenovane v inej brenchi

**Review by Kraci — APPROVED** [review]

**Pull request merged**